docs/mmteb/points/scores_from_old_system.jsonl

```diff
@@ -29,4 +30,4 @@
 {"GitHub": "isaac-chung", "New dataset": 36, "Review PR": 12}
 {"GitHub": "asparius", "New dataset": 8}
 {"GitHub": "rbroc", "New dataset": 12}
-{"GitHub": "dwzhu-pku", "New dataset": 12}
\ No newline at end of file
+{"GitHub": "dwzhu-pku", "New dataset": 12}
```

Comment:
I think that the last empty line here will make the table generation action fail, can you remove it please?